webcommon/lib.v8debug/src/org/netbeans/lib/v8debug/client/cmdline/V8Debug.java

```diff
@@ -1020,17 +1020,10 @@ private String printStr(V8Value value) {
                 if (o.getProperties() != null) {
                     Map<String, V8Object.Property> properties = o.getProperties();
                     String newLine = System.getProperty("line.separator");
-                    for (String propName : properties.keySet()) {
-                        sb.append(newLine);
-                        sb.append("  ");
-                        sb.append(propName);
-                        sb.append(" = ");
-                        V8Object.Property property = properties.get(propName);
-                        sb.append('(');
-                        sb.append(property.getType());
-                        sb.append(") ref: ");
-                        sb.append(property.getReference());
-                    }
+
+                    properties.forEach((propName, v8Prop) -> {
+                        sb.append(newLine + "  " + propName + " = (" + v8Prop.getType() + ") ref: " + v8Prop.getReference());
+                    });
```

Comment:
this PR is not about `append`. Please keep the code as is and restrict changes to collections.